### PR TITLE
#1695 - Update documentation for deployment via systemd

### PR DESCRIPTION
**Pull request by jcklie**
**What's in the PR**
* Update documentation for deployment via systemd

**How to test manually**
* Build docu

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [x] PR updates documentation

### CODE REVIEW

**Comment by reckart** [comment]
Does that work? Don't we need to pass along `-Dinception.home=...` somehow?

**Comment by jcklie** [comment]
It still uses the `inception.conf`:

Create the file /srv/inception/inception.conf with the following content

JAVA_OPTS="-Djava.awt.headless=true -Dinception.home=/srv/inception"

File: inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc [hunk 1]
```diff
@@ -91,21 +98,21 @@ $ systemctl enable inception
 +
 [source,bash]
 ----
-$ service inception start
+$ systemctl inception start
```
Comment (reckart) [inline inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc:101]: When I try this, I get:

> Unknown operation inception

File: inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc [hunk 2]
```diff
@@ -73,11 +72,19 @@ $ chown www-data:www-data /srv/inception/inception.jar
...
+* Create a file in `/etc/systemd/system/inception.service` with the following content:
 +
 [source,bash]
 ----
-$ ln -s /srv/inception/inception.jar /etc/init.d/inception
+[Unit]
+Description=INCEpTION
+
+[Service]
+ExecStart= /srv/inception/inception.jar
```
Comment (reckart) [inline inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc:83]: Is the space between `=` and `/` intentional?

Comment (jcklie) [reply inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc]: Fixed

File: inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc (hunk 1 above)
Comment (jcklie) [reply inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc]: Fixed

**Pull request merged**